DESCRIPTION

```diff
@@ -20,7 +20,7 @@ Imports:
     geoknife (>= 1.2.0),
     sbtools (>= 1.0.2),
     streamMetabolizer (>= 0.9.7),
-    unitted (>= 0.2.6),
```

Comment:
I think we also now have newer min versions of dataRet and geoknife (the versions after their https switchovers)